ios/FluentUI/PersonaButton/PersonaButton.swift

```diff
@@ -82,14 +97,14 @@ public struct PersonaButton: View {
             Text(state.primaryText ?? "")
                 .lineLimit(1)
                 .frame(alignment: .center)
-                .scalableFont(font: tokens.labelFont)
-                .foregroundColor(Color(tokens.labelColor))
+                .font(.fluent(tokens.labelFont))
```

Comment:
Nit more related to fonts than with the PersonaButton:
I think "fluentFont" is a more descriptive name for these static methods that were created in the Font and UIFont extensions.

Reply:
Fair point! I'll spin up a separate PR to rename `fluentFont` and `resolveTokenModifier`.